pages/linux/fakeroot.md

```diff
@@ -0,0 +1,28 @@
+# fakeroot
+
+> Run a command in an environment, faking root privileges for file manipulation.
+> More information: <https://manpages.debian.org/latest/fakeroot/fakeroot.1.html>.
```

Comment:
Do you have any reason not to use manned.org?

Reply:
Because `manned.org` fetches it from the Debian Manual.

Reply:
Yeah, this is a debian project <https://tracker.debian.org/pkg/fakeroot>.

Reply:
Oh, I wasn't aware of that, only ever saw it on Arch.